@xen-orchestra/web-core/lib/components/counter/VtsCounter.vue

```diff
@@ -46,8 +46,8 @@ VARIANT + ACCENT
 .vts-counter {
   &.variant--primary {
     &.accent--brand {
-      --vts-counter--background-color: var(--color-normal-item-base);
-      --vts-counter--color: var(--color-normal-txt-item);
+      --vts-counter--background-color: var(--color-info-item-base);
+      --vts-counter--color: var(--color-info-txt-item);
```

Comment:
the accent `info` has been added in the DS for this component.
And that makes sense since we now have `brand` and `info` as color possibilities.